packages/server/src/modules/db/migrate.ts

```diff
@@ -0,0 +1,40 @@
+import * as migrate from 'migrate';
+import { MigrationSet } from 'migrate';
+
+class MockFileStore {
+  save(set: MigrationSet, cb: (err: Error | null) => void) {
+    cb(null);
+  }
+  load(cb: (...args: any[]) => void) {
+    cb(null, { migrations: [] });
+  }
+}
+
+/**
+ * Used to run migrations on test databases.
+ * These migrations are temporary.
+ */
+export function runContainerMigrations() {
```

Comment:
Here's an entrypoint to pass migrations file name (or directory)

Reply:
This isn't used for running migrations actually, just for testing purpose.